hoot-core/src/main/cpp/hoot/core/conflate/polygon/MultiPolygonCreator.cpp

```diff
@@ -110,7 +110,7 @@ Geometry* MultiPolygonCreator::_addHoles(vector<LinearRing*> &outers,
       // containing polygon we've found so far.
       else if (polygonIndex == -1 || (outerArea[j] < outerArea[polygonIndex]))
       {
-        auto_ptr<IntersectionMatrix> im(tmpPolygons[j]->relate(inners[i]));
+        boost::shared_ptr<IntersectionMatrix> im(tmpPolygons[j]->relate(inners[i]));
```

Comment:
Removing the `auto_ptr` is a good thing.  Why was it even an `auto_ptr` here?  Not even necessary.